code/controllers/subsystem/vote.dm

```diff
@@ -154,34 +142,38 @@ SUBSYSTEM_DEF(vote)
 	return .
 
 /datum/controller/subsystem/vote/proc/submit_vote(vote)
-	if(mode)
-		if(CONFIG_GET(flag/no_dead_vote) && usr.stat == DEAD && !usr.client.holder)
-			return FALSE
-		if(!(usr.ckey in voted))
-			if(vote && 1<=vote && vote<=choices.len)
-				voted += usr.ckey
-				choices[choices[vote]]++ //check this
-				return vote
-	return FALSE
+	if(!mode)
+		return FALSE
+	if(CONFIG_GET(flag/no_dead_vote) && usr.stat == DEAD && !usr.client.holder)
+		return FALSE
+	if(!(vote && 1<=vote && vote<=choices.len))
+		return FALSE
+	// If user has already voted
+	if(usr.ckey in voted)
+		choices[choices[choice_by_ckey[usr.ckey]]]--
+	else
+		voted += usr.ckey
+
+	choice_by_ckey[usr.ckey] = vote
+	choices[choices[vote]]++	//check this
```

Comment:
1. No alignment indentation
2. Check what? This comment makes no sense.

Reply:
1. what do you mean
2. this comment exists in the codebase already

Reply:
1. https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#tabs-not-spaces -- Use one tab to prefix a comment. As in, `code // comment`, never `code         // comment`.
2. Remove it 😛